src/resolvers/snapshot.ts

```diff
@@ -1,8 +1,8 @@
+import snapshot from '@snapshot-labs/snapshot.js';
 import axios from 'axios';
 import { getUrl, resize } from '../utils';
 import { max } from '../constants.json';
 import { fetchHttpImage, axiosDefaultParams } from './utils';
-import { getChecksummedAddress } from '../utils';
```

Comment:
We are not using getChecksummedAddress anywhere, maybe we can delete it from utils